gobblin-modules/gobblin-kafka-09/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSimpleStreamingExtractor.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gobblin.source.extractor.extract.kafka;
+
+import avro.shaded.com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Optional;
+import java.io.IOException;
+import java.nio.channels.ClosedChannelException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+
+import com.google.common.base.Preconditions;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+
+import gobblin.configuration.WorkUnitState;
+import gobblin.configuration.State;
+import gobblin.metrics.kafka.KafkaSchemaRegistry;
+import gobblin.metrics.kafka.SchemaRegistryException;
+import gobblin.source.extractor.DataRecordException;
+import gobblin.source.extractor.Watermark;
+import gobblin.source.extractor.WatermarkSerializerHelper;
+import gobblin.source.extractor.extract.EventBasedExtractor;
+import gobblin.source.extractor.StreamingExtractor;
+import gobblin.source.extractor.RecordEnvelope;
+import gobblin.source.extractor.CheckpointableWatermark;
+import gobblin.source.extractor.ComparableWatermark;
+import gobblin.source.extractor.extract.LongWatermark;
+import gobblin.metrics.Tag;
+import gobblin.writer.WatermarkStorage;
+
+
+/**
+ * An implementation of {@link StreamingExtractor} from which reads from Kafka and returns records as an array of bytes.
```

Comment:
Drop "from" from the comment. 
* An implementation of {@link StreamingExtractor} **from** which reads from Kafka and returns records as an array of bytes.